coops-ui/src/mixins/director-mixin.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Vue } from 'vue-property-decorator'
+/**
+ * Mixin that provides some utilities for director data.
+ */
+@Component({})
+export default class DirectorMixin extends Vue {
+  /**
+   * Sorter function based on ES6 map reduce to sort the list of directors in the json.
+   * @param fields The list of fields based on which director list needs to be sorted.
+   * @return The sort function
+   */
+  fieldSorter (fields: Array<string>): any {
```

Comment:
See if you can return a type different than "any" (for future strict TS).

Also, my linter is warning me that 'a' and 'b' implicitly have "any" type (which may also cause issues with future strict TS).